cmd/proxy/main.go

```diff
@@ -63,7 +66,7 @@ func main() {
 		maxQueueLength = maxConcurrency * 10
```

Comment:
nit: I think this flag should be required too (since we will set a default from outside the container)